docs/development/maintenance.md

```diff
@@ -13,104 +15,124 @@ modules of ILIAS.
-  An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"tester"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
-* **"testcase_writer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
+# Authorities
+The ILIAS-Community strives to create and maintain a secure, reliable, and
+adaptable learning management. We foster participation by a diverse set of
+developers, designers, testers and other contributors, but we also have to guarantee the sustainability and the quality
+of the ILIAS source code.
+
+To make sure people with diverse backgrounds and capabilities can participate
+in our Community and contribute to the development of ILIAS and its code base,
+we split the code into units (often called components, even though the term
+is hard to define) and we define a set of authorities community members can have
+concerning these units of code. We understand an authority as the counterpart of
+a responsibility: The people having the authorities to do something in a unit of
+code also assume the responsibility for the corresponding functions.
+
+For the context of ILIAS, we define **four** different authorities:
+
+1. **Authority to Sign of Code Changes**
+2. **Authority to Sign of Conceptual Changes**
+3. **Authority to Sign of Testcase Changes**
+4. **Authority to (De-)Assign Authorities**
+
+Each of these authorities can be held by a different set of people. This means
+that the social organisation of different groups working on different parts of
+the Code of ILIAS can be different.
+Right now ILIAS knows a few different social structures for the maintenance of
+units in the code of ILIAS:
+
+* In the **"Classic Model"** all authorities are concentrated in one person and
+this person works mostly alone.
+* In the **Coordinator Model** all authorities are concentrated in one or more
+people and they work together with other developers in the community to improve
+the code.
+* In the **"Test and Assessment Model"** the authorities **to Sign of Conceptual
+Changes**, **to Sign of Testcase Changes**, and **to (De-)Assign Authorities**
+lies with one person and the **Authority to Sign of Code Changes** with two
+others.
+
+More will surely emerge as the optimal solution for each unit is found.
+
+# Responsibilites
+Independently of the social organisation for each unit of code the following
+responsibilites need to be assumed:
+
+* All people holding an authority must agree to coordinate the development
+of their unit of code with the product manager and with the people maintaining
+other units of code.
+* One of the people holding either the **Authority to Sign of Code Changes** or
+the **Authority to Sign of Conceptual Changes** gets assigned related bugs
+automatically by the [Issue-Tracker](https://mantis.ilias.de). S/he is responsible
```

Comment:
„S/he is“ → „They are“?

Reply:
Aber ggf. nicht klar, dass dann nur die _eine_ Person gemeint ist.

Reply:
Hier würde ich es zur Klärung absichtlich nicht machen. "One of the people -> they" bezieht sich dann they auf die eine oder alle? Ich glaube der Satz ist auf beide Arten verständlich (es gäbe hier eine interessante Diskriminierungsdiskussion zu führen, aber lieber bei einem Bier ;-)).